blockchains/base/assets/0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA/info.json

```diff
@@ -1,17 +1,14 @@
 {
     "name": "USD Base Coin",
-    "website": "https://www.centre.io",
+    "symbol": "USDC",
+    "type": "BASE20",
```

Comment:
```suggestion
    "type": "BASE",
```

Reply:
@vcoolish in database we have BASE20 for now. Should we rename it to BASE ?

Reply:
@defisaur yes, please

Reply:
@defisaur Yes, please